JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/api/HttpServer.java

```diff
@@ -28,14 +45,44 @@
 import java.util.stream.Stream;
 
 import static ch.megard.akka.http.cors.javadsl.CorsDirectives.cors;
+import static com.softwaremill.session.javadsl.SessionTransports.CookieST;
 
-public class HttpServer extends AllDirectives {
+public class HttpServer extends HttpSessionAwareDirectives<UserSession> {
 
     private static final Logger LOGGER = LogManager.getLogger(HttpServer.class);
 
+    private static final String SECRET = "c05ll3lesrinf39t7mc5h6un6r0c69lgfno69dsak3vabeqamouq4328cuaekros401ajdpkh60rrt";
```

Comment:
Should this be configurable?

Reply:
Yes we I need to make that configurable.